sysbuild/Kconfig.xip

```diff
@@ -1,18 +1,21 @@
-# Copyright (c) 2023 Nordic Semiconductor
+# Copyright (c) 2023-2024 Nordic Semiconductor
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
 config SUPPORT_QSPI_XIP
 	bool
+	default y if SOC_NRF52840
```

Comment:
It doesn't really work on that SoC (see errata)

Reply:
I was the one that discovered and reported that issue many years ago :) though I thought it was said recently it works OK so long as you never relocated interrupt vectors, @anangl can you confirm?

Reply:
@anangl Can you give a judge?

Reply:
> It doesn't really work on that SoC (see errata)

What specific erratum do have in mind?

> I was the one that discovered and reported that issue many years ago :) though I thought it was said recently it works OK so long as you never relocated interrupt vectors, @anangl can you confirm?

It worked for me without any issues when I tried it some time ago, although I cannot I say that I tested it extensively.

Reply:
Had a look back at cases and it was when using data from XIP addresses as variables, so maybe not an issue after all.